src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url("./images/star.svg") no-repeat center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(-n+0),
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  height: 16px;
```

Comment:
you don't need set height and width, because it is already exist in .stars__star

Reply:
<img width="960" alt="image" src="https://github.com/mate-academy/layout_stars/assets/129754902/a5aeee26-e8a7-482d-867a-adca53a46ae4">

Reply:
After any changes, I write these commands as in the instructions.
<img width="625" alt="image" src="https://github.com/mate-academy/layout_stars/assets/129754902/58f7391a-4f66-400a-a48a-be3b10ea4e3f">